docs/docsite/rst/porting_guides/porting_guide_2.5.rst

```diff
@@ -204,6 +204,11 @@ desired.
     source files must be utf-8 encoded".  On Python3, the module will first attempt to pass the
     non-utf8 characters through verbatim and fail if that does not succeed.
 
+* The ``lineinfile`` module was changed to show a warning when using an empty string as a regexp.
```

Comment:
Worth stating which version of Ansible this changed in, as porting guides are generally from previous to 2.5.0, where as this change is part way through the 2.5.x cycle